### PR TITLE
adding getting started guide to the docs

### PR DESCRIPTION
**Pull request by dustinblack**
## Changes introduced with this PR

New "Getting Started" guide added to the core documentation.

---
By contributing to this repository, I agree to the [contribution guidelines](https://github.com/arcalot/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: docs/arcaflow/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,249 @@
+# Arcaflow Getting Started Guide
+
+## Introduction
+
+[Arcaflow](https://arcalot.io/arcaflow/) is a modular system that enables engineers to easily build complex parallelized workflows for test automation without requiring any pre-installation or deployment of prerequisite software stacks. It uses an **engine** and **plugin** architecture, together with which **workflows** are constructed. Workflows are composable and highly portable, capable of orchestrating complex interrelated actions between plugins. This effectively allows for engineering workflow expertise to be packaged up, version controlled, and shared for repeatability across environments and platforms.
```
[the remark this answers is not in the document]
Comment (portante) [reply docs/arcaflow/getting-started.md:5]: The comma (",") in "architecture, together with which" seems misplaced.

File: docs/arcaflow/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+ 
+The engine component is intended to run from your laptop, jump host, or wherever you have network connectivity to the System Under Test (SUT). No installation is required of any Arcaflow components, either on your system or the SUT. The engine processes the workflow definition and passes the required data to the plugins.
```
Comment (portante) [inline docs/arcaflow/getting-started.md:7]: s/System Under Test (SUT)/target environment/?

Comment (portante) [reply docs/arcaflow/getting-started.md:7]: Perhaps you'd want to say that "the engine invokes the plugins in the target environment according to the workflow definition and passes the required data to the plugins as directed by the workflow" ... or something like that.

**Review by portante — CHANGES_REQUESTED** [review]
Looks good.

File: docs/arcaflow/getting-started.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (dustinblack) [reply docs/arcaflow/getting-started.md:5]: I disagree about the comma, but it's pretty subjective.

**Review by portante — CHANGES_REQUESTED** [review]

**Review by jaredoconnell — CHANGES_REQUESTED** [review]

**Review by portante — APPROVED** [review]

**Review by tsebastiani — APPROVED** [review]

**Review by jaredoconnell — APPROVED** [review]

**Pull request merged**